sql/01-create-table-student.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE student_info.student(
+  id INT PRIMARY KEY NOT NULL,
```

Comment:
I don't know, but all the examples I've seen for ids used it. And I did want them to just be numbers.